### PR TITLE
ENYO-6257: RadioItem: background-color doesn't match GUI spec

### PR DESCRIPTION
**Pull request by Sangwook-Lee**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [ ] Documentation was added or is not needed

* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
As the GUI specification has been updated.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
I've updated the colors.

### Links
[//]: # (Related issues, references)
ENYO-6257

### CODE REVIEW

File: packages/moonstone/styles/colors-light.less [hunk 1]
```diff
@@ -199,8 +199,8 @@
 
 // RadioItem
 // ---------------------------------------
-@moon-radio-item-indicator-color: @moon-dark-gray;
-@moon-radio-item-indicator-bg-color: @moon-white;
+@moon-radio-item-indicator-color: #333333;
+@moon-radio-item-indicator-bg-color: fade(#888888, 50%);
 @moon-radio-item-focus-indicator-color: @moon-spotlight-text-color;
 @moon-radio-item-focus-indicator-bg-color: fade(#a6a6a6, 50%);
```
Comment (ryanjduffy) [inline packages/moonstone/styles/colors-light.less]: The specs seems to use the same bg color for focus as well so I think we should update this to `#888888` as well.

Comment (Sangwook-Lee) [reply packages/moonstone/styles/colors-light.less]: You meant `@moon-radio-item-focus-indicator-bg-color: fade(#888888, 50%);`  ?

Comment (ryanjduffy) [reply packages/moonstone/styles/colors-light.less]: Yes

Comment (Sangwook-Lee) [reply packages/moonstone/styles/colors-light.less]: Done!

**Pull request merged**